dirsrvtests/tests/suites/subentries/subentries_test.py

```diff
@@ -0,0 +1,201 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2021 RED Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ----
+
+# Author: Anton Bobrov <abobrov@redhat.com>
+
+import logging
+import pytest
+import os
+import ldap
+from ldap.controls import LDAPControl
+from lib389.utils import *
+from lib389._constants import *
+from lib389.topologies import topology_st as topo
+from lib389.idm.user import UserAccount, UserAccounts
+
+log = logging.getLogger(__name__)
+
+pytestmark = pytest.mark.tier0
```

Comment:
I'd also add a skipif mark for tests to avoid running it on versions that don't support this control (see https://github.com/389ds/389-ds-base/blob/master/src/lib389/lib389/rootdse.py).

Reply:
done

Reply:
I don't see the change, was the new commit pushed?